docs/src/api/class-locator.md

```diff
@@ -770,6 +770,13 @@ Returns the `node.textContent`.
 
 ### option: Locator.textContent.timeout = %%-input-timeout-%%
 
+## method: Locator.that
```

Comment:
This word was quite confusing to me until I saw the examples later on in the PR. It seems like this just means `and`. Let's consider renaming it to `and` since I think Boolean logic words will have more straightforward meaning/explanations.

Reply:
TBD at API review!